src/main/java/frc/team3388/robot/SystemFactory.java

```diff
@@ -31,6 +33,15 @@ public ExampleSystem createExampleSystem() {
         return new ExampleSystem(motor);
     }
 
+
+    public ShooterSystem shooterS() {
```

Comment:
The name is problematic.. it means nothing... 

Method names should illustrate an activity with a verb. Like `createShooterSystem`.